src/video/SDL_video.c

```diff
@@ -2366,6 +2366,18 @@ SDL_GetWindowBordersSize(SDL_Window * window, int *top, int *left, int *bottom,
     return _this->GetWindowBordersSize(_this, window, top, left, bottom, right);
 }
 
+void
+SDL_GetWindowDrawableSize(SDL_Window * window, int *w, int *h)
+{
+    CHECK_WINDOW_MAGIC(window,);
+
+    if (_this->GetWindowDrawableSize) {
```

Comment:
To possibly simplify the driver functions:
```suggestion
    int filler;
    CHECK_WINDOW_MAGIC(window,);


    if (w == NULL) {
        w = &filler;
    }
    if (h == NULL) {
        h = &filler;
    }

    if (_this->GetWindowDrawableSize) {
```
We do this sometimes but not always, not sure if we have a policy for filler variable like this.

Reply:
That makes sense! I actually started with this but noticed most existing driver implementations handle it, but I do like this in practice & will update.